examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    sim.advance_time(3600 * 24 * 30, 10, constraint_function=constraint_func_TID_failure)
</pre><p>It runs longer now! :) However, differently on my machine. Can you just add a np.random.seed(42) in the top somewhere? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>